server/views/pages/goal/update/review.njk

```diff
@@ -113,7 +113,7 @@
         <div class="govuk-form-group govuk-!-margin-bottom-4">
           {{ govukButton({
             type: "a",
-            href: "/plan/" + prisonerSummary.prisonNumber + "/goals/" + data.reference + "/update",
+            href: "/plan/" + prisonerSummary.prisonNumber + "/goals/" + data.goalReference + "/update",
```

Comment:
Why do we need to make this change? Seems to be an unrelated change? Something to do with updating a goal? 🤔 
Are you saying its never worked? 😬